### ISSUE TITLE
WordPerfect for Windows installers don't exit when finished

### ISSUE THREAD

**Issue by emendelson** [otya128/winevdm#247]
I've been experimenting with WineVDM and the installers for WordPerfect for Windows; for example, this:

https://winworldpc.com/product/wordperfect/6x-win

The installers for WPWin 5.2 and 6.1 don't exit when finished installing, so I have to shut down otvdm.exe from the task manager.

WordPerfect won't start after the installation is finished, but that may be related to the problem with the installers, so I'll save that for a separate issue later on.

**Comment by cracyc** [comment]
This is yet another DDE issue when it tries to create the program manager items.

**Comment by cracyc** [comment]
Once it's installed it fails to run for the same reason as https://github.com/otya128/winevdm/issues/147 but with ShowWindow also.   Maybe there needs to be a lock that the parent task waits on until the child calls SendMessage or GetMessage?  Even working around that it still fails to work though.
